packages/vscode-extension/src/handlers.ts

```diff
@@ -965,6 +965,13 @@ export async function publishHandler(args?: any[]): Promise<Result<null, FxError
   return await runCommand(Stage.publish);
 }
 
+export async function publishInDeveloperPortalHandler(
+  args?: any[]
+): Promise<Result<null, FxError>> {
+  ExtTelemetry.sendTelemetryEvent(TelemetryEvent.PublishStart, getTriggerFromProperty(args));
```

Comment:
Do we need to use a new event? `Publish Start` is also used by publish action. We cannot tell the difference between them.